tests/test_migrate.py

```diff
@@ -35,6 +38,15 @@ def remote_client() -> QdrantClient:
     client.close()
 
 
+@pytest.fixture
+def second_remote_client() -> QdrantClient:
+    client = QdrantClient(port=6334)
+    delete_collections(client)
+    yield client
+    delete_collections(client)
+    client.close()
```

Comment:
we have only one running qdrant instance in the CI, thus, the second remote client will connect to the same database, which does not make sense